assets/src/dashboard/theme.js

```diff
@@ -107,6 +121,9 @@ const theme = {
       letterSpacing: '0.01em',
     },
   },
+  zIndex: {
+    popoverMenu: 10,
```

Comment:
IMHO `zIndex` doesn't fit very well with themes. Let's move to constants.

Reply:
👍 If that's preferred by your team I'll go ahead and move it. In previous projects I've leaned towards adding it in the theme because it's pertaining to style implementation.